members/models.py

```diff
@@ -32,6 +33,7 @@ class Member(models.Model):
     uio_username = models.CharField(max_length=15, null=True, blank=True)
     comments = models.TextField(null=True, blank=True)
     last_edited_by = models.ForeignKey(User, related_name='modifier', null=True)
+    gdpr_approval = models.BooleanField(default=False, help_text='Privacy policy agreement')
```

Comment:
Antakeligvis bedre om vi hadde brukt en dato her, slik at man kan forstå når det skjedde.

Har vi egentlig noen policy liggende noe sted?

Viser forøvrig til diskusjon på Slack: https://cybernetisk.slack.com/archives/C03QDK7CV/p1547123717047400
Det ble aldri konkludert med hva som var det reelle behovet, og om det faktisk er nødvendig med samtykke. Og det fremkommer f.eks. ikke hva man i så fall samtykker til, og hvordan evt. samtykke kan trekkes tilbake og hva slags betydning det får.